### ISSUE TITLE
How can I redirect all traffic (from port 80) to ssl?

### ISSUE THREAD

**Issue by jamesgibson14** [arunoda/meteor-up-legacy#286]

**Comment by arunoda** [comment]
Install force-ssl meteor package.

**Comment by ghgr** [comment]
Has someone tried this? I tried to force ssl in the example meteor app and when visiting http://url.com it does NOT redirect to https://url.com (SSL is working fine if I type https, but the redirection is not working).

I just made:
meteor create testApp
cd testApp
meteor add force-ssl
...configured mup.json with ssl (cert.pem)...
mup setup && mup deploy

I get no errors in setup nor in deployment, but the redirection from http to https just doesn't work

**Comment by TimFletcher** [comment]
I've just tried this and it worked for me.

My original config was without SSL so I added the certificate as per the instructions, deployed, reloaded and it would work with _both_ http and https. Adding `meteor add force-ssl` and redeployed. Now all requests to http:// are redirected to https://

The site is http://linkmetrics.io is you want to look at the headers etc. There are [some headers](http://meteorpedia.com/read/SSL) that need setting for force-ssl to work. I didn't have to do any of this but it may help you debug.
